cpp/src/nvgraph/include/Louvain_matching.cuh

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
```

Comment:
I would highly recommend moving Louvain files to cuGraph at this point (ie. outside of nvGraph). It should be fairly quick as they are not tightly connected to nvGraph and will help to transfer the clustering code to cuGraph as a native feature.